src/Cargo/sdk/Sdk.props

```diff
@@ -6,7 +6,8 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <UsingMicrosoftCargoSdk>true</UsingMicrosoftCargoSdk>
-    <MSBuildAllProjects Condition="'$(MSBuildToolsVersion)' != 'Current'">$(MSBuildAllProjects);$(MsBuildThisFileFullPath)</MSBuildAllProjects>
+    <MSBuildAllProjects Condition="'$(MSBuildToolsVersion)' != 'Current'">
```

Comment:
`MSBuildAllProjects` Is not needed anymore (unless you wanted to add a _different_ file to it. It's automatically produced by the engine and optimized. Just delete it.